flycheck.el

```diff
@@ -5493,6 +5506,11 @@ non-nil."
   (when flycheck-display-errors-function
     (funcall flycheck-display-errors-function errors)))
 
+(defun flycheck-clear-displayed-errors ()
+  "Clear errors using `flycheck-clear-displayed-errors-function'."
+  (when flycheck-clear-displayed-errors-function
```

Comment:
Do we really need this to be customizable? Why would users want a different behaviour?

Reply:
Not all error messages are displayed the same way. There are many packages that adjusts how the messages are displayed, so use childframes, others use overlays. Displaying and clearing messages have to be a symmetric operation operating on the same data type in order to be reliable.

Reply:
Also, see a usage example here

https://github.com/flycheck/flycheck-inline/pull/22/files